edxval/tests/test_api.py

```diff
@@ -1304,18 +1304,14 @@ def test_external_video_transcript(self):
         with self.assertRaises(VideoTranscript.DoesNotExist):
             VideoTranscript.objects.get(video_id=external_video_id)
 
-        api.import_from_xml(xml, external_video_id, external=True)
+        api.import_from_xml(xml, '')
         self.assert_transcripts(external_video_id, [self.transcript_data1, self.transcript_data2])
 
     def test_external_no_video_transcript(self):
         """
         Verify that transcript import for external video working as expected when there is no transcript.
         """
-        api.import_from_xml(
-            etree.fromstring('<video_asset/>'),
-            'does_not_exist_id',
-            external=True
-        )
+        api.import_from_xml(etree.fromstring('<video_asset/>'), '')
```

Comment:
do we need this test when we don't have external ? If yes then we need to change the test name.

Reply:
This test is only needed for external videos.